dsl/Prop.cc

```diff
@@ -11,12 +11,7 @@ using namespace std;
 
 namespace sorbet::dsl {
 
-optional<ChalkODMProp::NodesAndProp> ChalkODMProp::replaceDSL(core::MutableContext ctx, ast::Send *send) {
-    if (ctx.state.runningUnderAutogen) {
-        // TODO(jez) Verify whether this DSL pass is safe to run in for autogen
-        return std::nullopt;
-    }
-
+vector<unique_ptr<ast::Expression>> replaceDSL(core::MutableContext ctx, ast::Send *send) {
```

Comment:
Another thing you migth want to do is keep this as a private static member of `class Prop`. As it's written currently, as long as someone forward declares it with the right signature, they'll be able to call this function as `sorbet::dsl::replaceDSL`.